Source/Core/Core/DSP/DSPCodeUtil.cpp

```diff
@@ -218,4 +218,44 @@ bool SaveBinary(const std::vector<u16>& code, const std::string& filename)
     return false;
   return true;
 }
+
+bool DumpDSPCode(const u8* code_be, int size_in_bytes, u32 crc)
+{
+  const std::string binFile =
+      StringFromFormat("%sDSP_UC_%08X.bin", File::GetUserPath(D_DUMPDSP_IDX).c_str(), crc);
+  const std::string txtFile =
```

Comment:
Code style: `bin_file` and `txt_file`. Might as well fix that when you're touching the code anyways.